### PR TITLE
Change IAB Dashboard and Student History page icons: blue icon on gra…

### PR DESCRIPTION
**Pull request by jtreuting**
…y background

image

### CODE REVIEW

File: webapp/src/main/webapp/src/styles.less [hunk 1]
```diff
@@ -1526,7 +1526,7 @@ wide-radio-group {
   vertical-align: middle;
 
   > svg {
-    width: 100%;
+    width: 96%;
```
Comment (esotrope) [inline webapp/src/main/webapp/src/styles.less]: why not scale proportionally?

File: webapp/src/main/webapp/src/styles.less [hunk 2]
```diff
@@ -1572,12 +1572,19 @@ user-group {
   min-width: 75px;
 }
 
+.icon-wrapper {
+  padding: 5px;
+  background-color: @gray-lightest;
+  border-radius: 6px;
```
Comment (esotrope) [inline webapp/src/main/webapp/src/styles.less]: there should be some vars to rely on here for padding and radius

File: webapp/src/main/webapp/src/styles.less (hunk 1 above)
Comment (jtreuting) [reply webapp/src/main/webapp/src/styles.less]: the height should be as well, but i forgot to go back and do that.  this was the only way I could get the icon to not overlap the background on the right.

File: webapp/src/main/webapp/src/styles.less (hunk 2 above)
Comment (jtreuting) [reply webapp/src/main/webapp/src/styles.less]: do all browsers (like IE 11 and MS Edge) respect styling custom elements?

Comment (esotrope) [reply webapp/src/main/webapp/src/styles.less]: hm good q. i know we have quite a few styles done that way already. if not an alternative is adding `host: {class: 'component-name'}` to the `@Component()` to give it a class to bind to. done that before too

**Pull request merged**